frontend/public/components/modals/column-management-modal.tsx

```diff
@@ -122,48 +124,46 @@ export const ColumnManagementModal: React.FC<
               t('public~The namespace column is only shown when in "All projects"')}
           </Alert>
         </div>
-        <div className="row co-m-form-row">
-          <div className="col-sm-12">
-            <span className="col-sm-6">
-              <label>
-                {t('public~Default {{resourceKind}} columns', { resourceKind: columnLayout.type })}
-              </label>
-              <DataList
-                aria-label={t('public~Default column list')}
-                id="defalt-column-management"
-                isCompact
-              >
-                {defaultColumns.map((defaultColumn) => (
-                  <DataListRow
-                    key={defaultColumn.id}
-                    onChange={onColumnChange}
-                    disableUncheckedRow={areMaxColumnsDisplayed}
-                    column={defaultColumn}
-                    checkedColumns={checkedColumns}
-                  />
-                ))}
-              </DataList>
-            </span>
-            <span className="col-sm-6">
-              <label>{t('public~Additional columns')}</label>
-              <DataList
-                aria-label={t('public~Additional column list')}
-                id="additional-column-management"
-                isCompact
-              >
-                {additionalColumns.map((additionalColumn) => (
-                  <DataListRow
-                    key={additionalColumn.id}
-                    onChange={onColumnChange}
-                    disableUncheckedRow={areMaxColumnsDisplayed}
-                    column={additionalColumn}
-                    checkedColumns={checkedColumns}
-                  />
-                ))}
-              </DataList>
-            </span>
-          </div>
-        </div>
+        <Grid hasGutter className="co-m-form-row">
```

Comment:
We lost the gutter when the columns are stacked.

<img width="520" alt="Screenshot 2025-04-30 at 10 03 04 AM" src="https://github.com/user-attachments/assets/8ae14bb8-4e18-4672-8564-aa498eda181e" />